build/yaml/crd/vpc/crd.nsx.vmware.com_subnets.yaml

```diff
@@ -91,10 +81,21 @@ spec:
                 x-kubernetes-validations:
                 - message: Value is immutable
                   rule: self == oldSelf
+              subnetDHCPConfig:
+                description: DHCP configuration for Subnet.
+                properties:
+                  mode:
+                    default: DHCPDeactivated
```

Comment:
Do we need to set default value in CRD definition?

Reply:
Removed, thanks!